unitTestSupportFunctions/calculateDrag.m

```diff
@@ -0,0 +1,30 @@
+%% Function documentation
+%
+% Calculate drag force on a single body with no-slip boundary conditions in a rectangular domain
+%
+%               Input :
+%           FComplete : The complete force vector
+%              fldMsh : Nodes and elements for the fluid mesh
+%              homDBC : The global numbering of the DOFs where homogeneous
+%                       Dirichlet boundary conditions are applied
+%          parameters : Flow parameters
+%
+%
+%              Output :
+%                  Fx : Force (drag) on the body in x-direction
+%
+%% Function main body
+function Fx = calculateDrag (FComplete, fldMsh, homDBC, parameters)
+
+    % Get node indices on the body
+    indices = getBodyNodes(fldMsh, homDBC);
+    
+    % Reshape the input forces to [Fx, Fy, Fz]
+    FComplete = -reshape(FComplete, [3, length(FComplete)/3])';
```

Comment:
These three lines of code can be condensed to Fx = sum(FComplete(1:noDimensions:end),1) * parameters.rho; which should some all forces in the x-direction given the residual force vector multiplied by the density.